addons/mail/static/src/web/chatter.xml

```diff
@@ -74,10 +74,8 @@
         </div>
         <t t-if="state.thread.composer.type">
             <t t-if="state.thread.composer.type === 'message'">
-                <div class="flex-shrink-0 px-3 pb-1 pt-2 text-truncate" style="margin-left:48px;">
-                    <em class="text-muted">
-                        To followers of:
-                    </em>
+                <small class="flex-shrink-0 px-3 pt-3 text-truncate" style="margin-left:48px;">
```

Comment:
If the purpose is purely stylistic, prefer to use the Bootstrap class `.small` rather than the `<small>` HTML element, as the latter conveys some specific semantics that are not relevant here.